<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>D:/Cricspoint/Cricspoint41-03.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Enter Player Details</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Birth Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Batting Hand</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bowling Hand</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="pTeam">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="bDate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="batr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">battingHand</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="batl">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">battingHand</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">bowlingHand</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="bowlr">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">bowlingHand</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>502</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Shocking Headline</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="battingHand"/>
  <buttongroup name="bowlingHand"/>
 </buttongroups>
</ui>
